README.md

```diff
@@ -1 +1,95 @@
-# javascript-capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+# HOLLYWOOD CENTRAL, a JavaScript capstone project 
+
+> It's a responsive web application based on externals APIs, which randomly displays many TV Shows, Actors and Actresses allowing the user to request details of each one through pop-up windows and give them "likes" and comments
+
+## Live Demo
+
+[Live Demo Link](https://lucascomamala.github.io/javascript-capstone-project/)
```

Comment:
- [ ] Ad the previous reviewer mentioned kindly consider showcasing a piece of code your pride of in your video presentation